src/configsystem.rs

```diff
@@ -11,6 +11,10 @@ pub struct Config {
     pub export_step: i64,
     pub export_directory: String,
     pub export_file_name_prefix: String,
+    #[serde(default)]
+    pub export_system_state: bool,
+    #[serde(default)]
+    pub export_body_history: bool,
```

Comment:
I'm not sure how the `#[serde(default)]` macro works, so I'll check that out. I'm wondering how it knows what the default is set to.

Reply:
https://serde.rs/field-attrs.html#default

> If the value is not present when deserializing, use the Default::default().

Ah, okay, so since it's a `bool`, the default for `bool`s is always false unless otherwise specified, I'm guessing

Reply:
https://doc.rust-lang.org/std/primitive.bool.html#impl-Default-for-bool
Yes!

Reply:
Yes, we could also supply a function that returns a default value, which might be interesting for numbers or vectors.